docs/UserGuide.md

```diff
@@ -99,10 +99,8 @@ delete the user that is currently displayed.
 Adds a book to the library.
 
 Format: `add book t/TITLE a/AUTHOR`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0).
-</div>
+* The title **must only contain alphanumerical characters and spaces**
+* The author **must only contain alphabetical characters and spaces**
```

Comment:
Are we going to change this to allow author to be alphanum as well?

Reply:
So sorry! I kind of forgot if the TA allowed this to be considered a bug. I took the safer side and decided not to change the regex as it might be considered a feature enhancement (a similar issue can be found in the Forums where the Prof mentioned this type of change to be a feature enhancement which should not be allowed). I can check with the TA again, but if we cannot change this (eg. deemed as a feature enhancement), I will take responsibility for this.

Reply:
Okay, I have confirmed with the TA; this can be considered as a bug since we do expect users to pass numbers into the Author field. I will change the details accordingly.